test/mongodb.test.js

```diff
@@ -701,6 +718,34 @@ describe('mongodb connector', function() {
     });
   });
 
+  it('should properly retrieve embedded model properties', function(done) {
+    const data = {name: 'Mitsos', location: {city: 'Volos', country: 'Greece'}};
+    WithEmbeddedProperties.create(data, function(err, createdModel) {
+      if (err) return done(err);
+      WithEmbeddedProperties.findById(createdModel.id, function(err, dbModel) {
+        if (err) return done(err);
+        const modelObj = dbModel.toJSON();
+        const dataObj = Object.assign({id: modelObj.id}, data);
+        modelObj.should.be.eql(dataObj);
```

Comment:
Can we use `deepEqual` for object property values here i.e. `modelObj.should.deepEqual(dataObj);`

Reply:
`eql` is the same with `deepEqual` and is also used in more places in the file. Would you like to change it to the `deepEqual` you suggest ?

Reply:
In that case, fine by me to keep it as `eql` 👍

Reply:
OK cool! Thanks! I am resolving this so the PR moves on